src/cares_wrap.cc

```diff
@@ -466,7 +466,11 @@ void cares_wrap_hostent_cpy(struct hostent* dest, struct hostent* src) {
   dest->h_addrtype = src->h_addrtype;
 }
 
-class QueryWrap;
+template <void (*F)(ares_channel, const char *, int, int, ares_callback, void *)>
```

Comment:
Style nit: Left-leaning pointers (e.g. `const char*`, not `const char *`, here and below)

If you’re unsure about something, we have a [C++ style guide](https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md), and, very recently, introduced `make format-cpp CLANG_FORMAT_START=upstream/master` (you may need to run `make format-cpp-build` first)

Reply:
@addaleax thanks for the tip! this produces `tools/clang-format/node_modules/` though, in my working directory. Good idea to add this to `.gitignore` in this PR? or should that be a different PR by itself?